modules/lobby/src/main/LobbySocket.scala

```diff
@@ -96,10 +97,9 @@ final class LobbySocket(
         send(P.Out.tellSri(hook.sri, gameStartRedirect(game pov creatorPlayerIndex)))
         send(P.Out.tellSri(sri, gameStartRedirect(game pov !creatorPlayerIndex)))
 
-      case JoinSeek(userId, seek, game, creatorPlayerIndex) =>
+      case JoinSeek(userId, _, game, creatorPlayerIndex) =>
         lila.mon.lobby.seek.join.increment()
-        send(Out.tellLobbyUsers(List(seek.user.id), gameStartRedirect(game pov creatorPlayerIndex)))
-        send(Out.tellLobbyUsers(List(userId), gameStartRedirect(game pov !creatorPlayerIndex)))
+        send(RemoteSocket.Protocol.Out.tellUser(userId, gameStartRedirect(game pov !creatorPlayerIndex)))
```

Comment:
I feel this is not appropriate to import a socket from another socket but on LILA-WS side the 
```
    case TellLobbyUsers(us, json) =>
      users.tellMany(us, ClientIn.onlyFor(_.Lobby, ClientIn.Payload(json)))
```
seem to not be working as expected, maybe because the user is on the Correspondence tab instead of the Lobby tab ?
What do you suggest ? :)

Reply:
have you looked at the lichess code base to see if they have added some code that would help in this case?
If that case is skipping correspondence users then it would be better to create a new case for this, have you followed the code to see why it might not be working?

Reply:
> have you looked at the lichess code base to see if they have added some code that would help in this case?
yes but it could be related to another file than the (obivous) ones I checked

> have you followed the code to see why it might not be working?
 Tried but the socket events are always a bit messy at first
 
 I can keep searching this afternoon for example, but wanted to make this checkpoint in case it would have been already too much time invested.

Reply:
Also this code handles both the teal-time and correspondence games. 
- For real-time games we want to redirect both users
- For correspondence we want to only redirect the user that clicked on it

Reply:
would also be happy not redirecting the user to the game, but reload the page/removing the correspondence seek, and hopefully seeing the appear in games in progress, redirecting to that tab could be useful instead.